hack/lib/init.sh

```diff
@@ -35,6 +36,7 @@ API_PACKAGES="\
 github.com/openshift/api/apps/v1,\
 github.com/openshift/api/authorization/v1,\
 github.com/openshift/api/build/v1,\
+github.com/openshift/api/console/v1,\
```

Comment:
Not sure if this should be in `API_PACKAGES`. @bparees do you know?

Reply:
i don't see any of the other CRD-related packages listed, so probably not.

Reply:
removed.